Resources/public/css/styles.css

```diff
@@ -216,6 +216,11 @@ body.fixed .content-header .navbar {
     padding: 15px;
 }
 
+.sidebar-menu li.keep-open > .treeview-menu {
+    display: block !important;
+    height: auto !important;
```

Comment:
Yes. Click please on other collapsed group arrow to open it. You will see collapsing animation on group with keep_open.